src/screens/Home/index.js

```diff
@@ -1,24 +1,30 @@
-import React from 'react';
+import React, { useContext, useCallback } from 'react';
 import i18next from 'i18next';
 
+import { AuthContext } from '@components/ProtectedRoute';
 import logo from '@assets/logo.svg';
 import styles from './styles.module.scss';
 
 function Home() {
+  const context = useContext(AuthContext);
```

Comment:
you can do `const { handleSetIsAuth, ... } = useContext(AuthContext);`, I like it more than getting the whole context. Yet another subjective thing